tcs-service/src/main/resources/config/application-local.yml

```diff
@@ -21,6 +21,8 @@ spring:
 
 logging:
     file: ${LOG_DIR:${HOME}}/tcs.log
+    level:
+      root: debug
```

Comment:
Even running locally, logging "debug" by default causes a lot of noise and can make it harder to see application specific logs.
Did you want to set this as the default for TCS?

Reply:
Just updated it for some debug level logging, I'm okay to change it back.